### ISSUE TITLE
Typography fixes

### ISSUE THREAD

**Issue by inderpreetsingh** [BRL-CAD/OGV#69]
There are bunch of typography design errors in OGV that needs to be solved

Todo coming soon!

**Comment by DeekshithSuresh** [comment]
Sir Can i take up these issues in order to solve it

**Comment by brlcad** [comment]
@DeekshithSuresh Of course.  You don't need permission.  Submitting a pull request with your prospective changes will get attention.
